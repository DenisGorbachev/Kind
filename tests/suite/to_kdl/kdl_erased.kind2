#kdl_name = A
Ata : U60
Ata = 2

#kdl_name = B
#kdl_erase
Be : U60
Be = 3

#kdl_name = C
Ce : U60
Ce = 4
