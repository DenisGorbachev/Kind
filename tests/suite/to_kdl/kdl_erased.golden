// Ata : U60
fun (A) {
  (A) = #2
}

// Ce : U60
fun (C) {
  (C) = #4
}

